designer/server/src/main/scala/pl/touk/nussknacker/ui/util/TapirCodecs.scala

```diff
@@ -0,0 +1,91 @@
+package pl.touk.nussknacker.ui.util
+
+import cats.implicits.toTraverseOps
+import pl.touk.nussknacker.engine.api.process.{ProcessName, VersionId}
+import pl.touk.nussknacker.ui.util.HeadersSupport.{ContentDisposition, FileName}
+import sttp.tapir.Codec.PlainCodec
+import sttp.tapir.CodecFormat.TextPlain
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, Schema}
+
+object TapirCodecs {
```

Comment:
Move here codecs from `ComponentApiEndpoints.ComponentCodec` as well. They are also general-purpose codes.

Reply:
`ComponentApiEndpoints.ComponentCode` is in `designer/restmodel` module. Should I move codecs to server module and leave the endpoints definition in the restmodel module?
If the endpoints definition stays, code will change to this:
```
  def componentsListEndpoint(implicit componentCodec: PlainCodec[DesignerWideComponentId]):
```
Where do you stand on this topic?

Reply:
Uh I missed that. Let it be as it is